bin/setup

```diff
@@ -42,6 +42,9 @@ FileUtils.chdir APP_ROOT do
   puts "== Install JavaScript dependencies =="
   system! "yarn install"
 
+  puts "== Install Python dependencies =="
+  system! "pip3 install --user opencv-python-headless"
```

Comment:
Developers should re-run `bin/setup` to make sure. this job can run with its needed deps.